### DISCUSSION TITLE
POM file error after importing maven project

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#32] [Q&A] [answered]
Dear Team,

am getting below error in pdf export pom file. i know it is causing because of missing dependency and/or linking parent POM.
may it is simple issue for experts not for me. can you please help me.
![pdfExporter_POM_Error](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/45649812/ea59855d-646e-43ff-b439-0a7f45df2191)
![generic Parent pom](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/45649812/f170f580-86a9-4f66-b248-99eac048b824)
![pdfExporter_POM_Error](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/45649812/6d6ef92e-b80d-449f-92a7-50aecc1bbf1f)
 

Multiple markers at this line
- Project build error: Non-resolvable parent POM for ch.sbb.polarion.extensions:ch.sbb.polarion.extension.pdf-exporter:5.1.3-SNAPSHOT: The following artifacts could not be resolved: ch.sbb.polarion.extensions:ch.sbb.polarion.extension.generic.parent-pom:pom:4.10.0 (absent): ch.sbb.polarion.extensions:ch.sbb.polarion.extension.generic.parent-pom:pom:4.10.0 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM
- Non-resolvable parent POM for ch.sbb.polarion.extensions:ch.sbb.polarion.extension.pdf-exporter:5.1.3-SNAPSHOT: The following artifacts could not be resolved: ch.sbb.polarion.extensions:ch.sbb.polarion.extension.generic.parent-pom:pom:4.10.0 (absent): ch.sbb.polarion.extensions:ch.sbb.polarion.extension.generic.parent-pom:pom:4.10.0 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced

below error in ch.sbb.polarion.extension.generic.parent-pom

Multiple markers at this line
- com.polarion.alm.projects:projects:jar:2404 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced
- Missing artifact com.polarion.alm.projects:projects:jar:2404

thanks in advance

**Comment by ariwk** [comment]
Hi @ukrishne 

The missing artefacts are currently unavailable due to Siemens licensing restrictions. However, the artefacts generated by us will be published to Maven Central soon.

I will update the documentation and inform you as soon as possible. In the meantime, please refer to the template at [CONTRIBUTING.md](https://github.com/SchweizerischeBundesbahnen/open-source-polarion-java-repo-template/blob/main/CONTRIBUTING.md#tools-and-packages). Please note that the repository containing the Python script is not yet public, as it is still pending review.

Regards, Adam

Refs:  #39

**Comment by ukrishne** [comment]
thanks adam for your response. I will wait for your next update.

**Comment by grigoriev** [comment, marked as answer]
Hi @ukrishne,

the [polarion-artifacts-deployer](https://github.com/SchweizerischeBundesbahnen/polarion-artifacts-deployer/ ) is public now.

**Comment by ukrishne** [comment]
thank you so much

On Wed, Jul 3, 2024 at 8:51 PM Sergey Grigoriev ***@***.***>
wrote:

> Closed #32
> <https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/discussions/32>
> as resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/discussions/32>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AK4I7FGMU6QVUXANWHGO2SDZKQJHPAVCNFSM6AAAAABJVPZ7TWVHI2DSMVQWIX3LMV45UABFIRUXGY3VONZWS33OIV3GK3TUHI5E433UNFTGSY3BORUW63R3GEZTOMBWG43Q>
> .
> You are receiving this because you were mentioned.Message ID:
> <SchweizerischeBundesbahnen/ch.sbb.polarion.extension.
> ***@***.***>
>


-- 
Thanks & Regards
   Umesh
